components/kyma-environment-broker/internal/process/provisioning/ems_provision.go

```diff
@@ -1,7 +1,6 @@
 package provisioning
```

Comment:
I haven't expected that... Don't you think we should ask creators of this step for approval?

Reply:
I have received info that those steps are disabled and planned to be removed so that think there is point further verification.